_data/harnesses/generate-test-data-streams/ksql.yml

```diff
@@ -0,0 +1,198 @@
+          container: ksqldb-cli
+          docker_bootup_file: tutorial-steps/dev/start-cli.sh
+          stdout:
+            directory: tutorial-steps/dev/outputs
+          column_width: 20
+          render:
+            skip: true
+          stdin:
+            - file: tutorial-steps/dev/create-connector-02.sql
+              render:
+                file: tutorials/generate-test-data-streams/ksql/markup/dev/create-connector-02.adoc
+
+        - action: sleep
+          ms: 5000
+          render:
+            skip: true
+            
+    - title: Join the test data streams in ksqlDB
+      content:
+        - action: docker_ksql_cli_session
+          container: ksqldb-cli
+          docker_bootup_file: tutorial-steps/dev/start-cli.sh
+          stdout:
+            directory: tutorial-steps/dev/outputs
+          column_width: 255
+          render:
+            skip: true
+          stdin:
+            - file: tutorial-steps/dev/check-connector-02.sql
+              render:
+                file: tutorials/generate-test-data-streams/ksql/markup/dev/check-connector-02.adoc
+            - file: tutorial-steps/dev/join-streams-declare.sql
+              render:
+                file: tutorials/generate-test-data-streams/ksql/markup/dev/join-streams-declare.adoc
+            - file: tutorial-steps/dev/join-streams-do.sql
+              render:
+                file: tutorials/generate-test-data-streams/ksql/markup/dev/join-streams-do.adoc
+
+    - title: Clean up
+      content:
+      - action: execute
+        file: tutorial-steps/dev/clean-up.sh
+        render:
+          file: tutorials/generate-test-data-streams/ksql/markup/dev/clean-up.adoc
+
+
+prod:
+  steps:
+    - title: Write your statements to a file
+      content:
+        - action: make_file
+          file: src/statements.sql
+          render:
+            file: tutorials/generate-test-data-streams/ksql/markup/dev/make-src-file.adoc
+
+    - title: Send the statements to the REST endpoint
+      content:
+        - action: execute_async
+          file: tutorial-steps/dev/docker-compose-up.sh
+          render:
+            skip: true
+
+        - action: execute
+          file: tutorial-steps/dev/wait-for-containers.sh
+          render:
+            skip: true
+
+        - action: execute
+          file: tutorial-steps/prod/send-to-api.sh
+          stdout: tutorial-steps/prod/outputs/send-to-api.out
+          render:
+            file: tutorials/generate-test-data-streams/ksql/markup/prod/submit-to-api.adoc
+
+    - title: Validate the deployment
+      content:
+        - action: execute
+          file: tutorial-steps/prod/check-deploy.sh
+          stdout: tutorial-steps/prod/outputs/check-deploy.out
+          render:
+            file: tutorials/generate-test-data-streams/ksql/markup/prod/check-deploy.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/clean-up.sh
+          render:
+            skip: true
```

Comment:
Add CCloud section like the other tutorials?

Reply:
Voluble isn't available on CCloud

Reply:
It doesn't mean we can't run connector locally connected to cloud.

Reply:
@rmoff Voluble is a connector, so it can be run in a self-managed connect cluster to CCloud.

Reply:
¯\_(ツ)_/¯ I don't think it makes sense to crowbar in Cloud to every single Connector tutorial. It's a tutorial on how to use a connector, not run it end-to-end. Consider, we also don't include how to run it outside of Docker, because that's left to the user to figure out.